src/DebugHelper.ts

```diff
@@ -0,0 +1,56 @@
+import { ELM } from './types/ELMTypes';
+import { ValueSetMap } from 'cql-execution';
+import fs = require('fs');
```

Comment:
```suggestion
import fs from 'fs';
```

Reply:
Updated. You may have to clean away your branch and re-pull, since I had to amend the commit before this one.